wen-restart/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "solana-wen-restart"
+description = "Automatically repair and restart protocol"
+documentation = "https://github.com/solana-foundation/solana-improvement-documents/pull/46"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+publish = false
+
+[dependencies]
+log = { workspace = true }
+prost = { workspace = true }
+prost-types = { workspace = true }
+solana-gossip = { workspace = true }
+solana-ledger = { workspace = true }
+solana-logger = { workspace = true }
+solana-program = { workspace = true }
+solana-runtime = { workspace = true }
+solana-sdk = { workspace = true }
+solana-vote-program = { workspace = true }
+
+[dev-dependencies]
+serial_test = { workspace = true }
+solana-entry = { workspace = true }
+solana-streamer = { workspace = true }
+
+[build-dependencies]
+prost-build = "0.11.4"
```

Comment:
Might as well add this dependency to the workspace toml now, in case we use it anywhere else.

Reply:
Done.